client.go

```diff
@@ -58,21 +60,24 @@ type Client interface {
 }
 
 // NewClient creates and builds new stats client instance by given dsn
-func NewClient(dsn, prefix string) (Client, error) {
+func NewClient(dsn string) (Client, error) {
 	dsnURL, err := url.Parse(dsn)
 	if err != nil {
 		return nil, err
 	}
 
+	// do not care about parse error, as default value is set to false that is fine for us
+	unicode, _ := strconv.ParseBool(dsnURL.Query().Get("unicode"))
```

Comment:
I would definitely recommend to handle the error here, even if the input is coming from a "trusted source"

Reply:
The output of this function is always true or false and in case of the error it is false - that is fine actually